commons/pom.xml

```diff
@@ -101,11 +107,11 @@
                   <Export-Package>
                      ${project.groupId}.commons.*;version=${project.version};-split-package:=error
                   </Export-Package>
-                  
+
                   <Import-Package>
                   !sun.misc,
                   *
-                  
+
```

Comment:
This empty line looks weird, maybe it was a comment at some point and it got deleted by the release script?